run/ktor-hello-world/README.md

```diff
@@ -0,0 +1,22 @@
+Hello Kotlin Ktor
+-----------------
+
+## Run Locally:
+```
+./gradlew dev
+```
+
+## Click-to-Deploy on Google Cloud Run
+[![Run on Google Cloud](https://deploy.cloud.run/button.svg)](https://deploy.cloud.run)
+
+## CLI Deploy on Google Cloud Run
+1. [Install & setup gcloud](https://cloud.google.com/sdk/install)
```

Comment:
If you do this before using the click to deploy, you need to enable the Cloud Run API. I assume the button does it for you but I did not test this.

Reply:
Yeah, Cloud Run Button does enable the API for the user.